instrumentation/servlet/servlet-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/v5_0/dispatcher/RequestDispatcherAdviceHelper.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.v5_0.dispatcher;
+
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.javaagent.instrumentation.api.Java8BytecodeBridge;
+
+public class RequestDispatcherAdviceHelper {
+  public static Context getStartParentContext(Object servletContextObject) {
```

Comment:
This method needs documentation. I barely understand it and I am not sure I do it correctly.

Reply:
Added some documentation.